controllers/front/orderSuccess.php

```diff
@@ -251,8 +251,22 @@ private function displayOrderConfirmation($intent)
                 $id_cart = $intent->metadata->id_cart;
             }
 
+            ProcessLoggerHandler::logInfo(
```

Comment:
it is a little bit too much to add this debug for all marchand, isn't it ?

Reply:
I added this log to quickly identify if id_cart is a problem (empty for example) and to see which cart is linked to the order directly in the logs